pkg/controller/installation/core_controller.go

```diff
@@ -456,6 +456,14 @@ func (r *ReconcileInstallation) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	// A status is needed at this point for operator scorecard tests.
+	// status.variant is written later but for some tests the reconciliation
+	// does not get to that point.
+	instance.Status = operator.InstallationStatus{}
```

Comment:
This will overwrite any previously written status. Is that what we want here? Or should we be maintaining any status that the controller had previously written?

Reply:
No, that wasn't right. I've updated the commit so that we only do the pre-write of the installation status if there was no status at the start of the reconcile.